packages/mgt-components/src/components/mgt-tasks-base/mgt-tasks-base.ts

```diff
@@ -224,7 +224,7 @@ export abstract class MgtTasksBase extends MgtTemplatedComponent {
    * @memberof MgtTasksBase
    */
   protected clearState(): void {
-    this.clearNewTaskData();
+    this.clearNewTaskData;
```

Comment:
Without the parenthesis here the function is not called, you're just accessing the reference to the function. 

Given that the `clearNewTaskData` function is only calling `this.requestUpdate()` we can remove this line entirely.
```suggestion
```